shared/types/equipment.types.ts

```diff
@@ -1,14 +1,20 @@
-export enum EquipmentTypes {
-    WEAPON   = 'weapon',
-    SHIELD   = 'shield',
-    NECK     = 'neck',
-    WAIST    = 'waist',
-    HEAD     = 'head',
-    CHEST    = 'chest',
-    ARMS     = 'arms',
-    LEGS     = 'legs',
-    FEET     = 'feet',
-    RING     = 'ring',
-    EARRING  = 'earring',
-    TALISMAN = 'talisman'
+export enum ArmorTypes {
+    HEAD  = 'head',
+    UPPER = 'upper',
+    LOWER = 'lower'

```

Comment:
Food for thought : Maybe as extra option a full body armor type, or a distinction between heavy armor, light armor,  robes (for mage types)